modules/virtualisation/podman/podman.nix

```diff
@@ -0,0 +1,276 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.virtualisation.podman;
+  toml = pkgs.formats.toml { };
+  json = pkgs.formats.json { };
+
+  inherit (lib) mkDefault mkIf mkMerge mkOption types;
+
+  podmanPackage = (pkgs.podman.override { inherit (cfg) extraPackages; });
+
+  # Provides a fake "docker" binary mapping to podman
+  dockerAlias = pkgs.runCommandNoCC
+    "${podmanPackage.pname}-docker-alias-${podmanPackage.version}" {
+      outputs = [ "out" "man" ];
+      inherit (podmanPackage) meta;
+    } ''
+      mkdir -p $out/bin
+      ln -s ${podmanPackage}/bin/podman $out/bin/docker
+
+      mkdir -p $man/share/man/man1
+      for f in ${podmanPackage.man}/share/man/man1/*; do
+        basename=$(basename $f | sed s/podman/docker/g)
+        ln -s $f $man/share/man/man1/$basename
+      done
+    '';
+  podmactl = pkgs.callPackage ./podmactl { };
+
+  machineOpts = {
+    # options here are loaded into python
+    # for simplicity, please use snake_case
+    options = {
+      active = mkOption {
+        type = types.bool;
+        default = false;
+        description =
+          "This machine should be started. Only one machine can be active at a time";
+      };
+      qemu_binary = mkOption {
```

Comment:
nix options are typically written using camelCase, same for the rest of this file

Reply:
This is specifically because it's exported into a JSON which is used by the python script to apply the machine config. There's a comment a few lines up that mentions this.

I'll have a look at the rest of the file and see if I didn't it respect it outside of machine options.